distribution/lib/Standard/Base/0.0.0-dev/src/Internal/IO_Helpers.enso

```diff
@@ -0,0 +1,3 @@
+private
+
+println message ln = @Builtin_Method "IO.println"
```

Comment:
I assume a typo? There was `nl` and here is `ln`.

Reply:
renamed to `ends_with`.